edurange_refactored/react/pages/home/dashboard/instructor/src/InstructorDash.js

```diff
@@ -6,43 +6,81 @@ import axios from 'axios';
-  const [scenGroup_name_state, set_scenGroup_name_state] = useState('');
-  const [scenarioDetail_state, set_scenarioDetail_state] = useState({})
 
+  // CREATE NEW SCENARIO GROUP 
 
-  const handleRadioChange = (event) => {
-    set_scenarioType_state(event.target.value);
+  // State of desired name input field for adding NEW ScenarioGroup to db
+  // IMPORTANT! (not to be confused with the name of currently existing scen group that is used as arg for create_scenario!)
+  const [newScenGroup_name_state, set_newScenGroup_name_state] = useState('');
+  
+  const handle_newGroup_name_change = (event) => {
+    set_newScenGroup_name_state(event.target.value);
   };
-
-  const handle_scenNameChange = (event) => {
-    set_scenarioName_state(event.target.value);
+  const handle_createGroup_submit = (event) => {
+    event.preventDefault();
+    axios.post('/api/create_group', {
+      group_name: newScenGroup_name_state
+    })
+    .then(response => {
+      console.log('Group created:', response.data);
+      set_newScenGroup_name_state(''); // Reset the input field after submission
+    })
+    .catch(error => {
+      console.error('Error creating group:', error);
+    });
   };
-  const handle_groupNameChange = (event) => {
-    set_scenGroup_name_state(event.target.value);
+  // NEW SCENARIOS
+  // just for creating a new scenario, not updating
+  const [newScenType_state, set_newScenType_state] = useState('');
+  const [newScenName_state, set_newScenName_state] = useState('');
+  
+  // IMPORTANT! (not to be confused with desired name for NEW scen group that is used as arg for create_group!)
+  const [newScen_groupName_state, set_newScen_groupName_state] = useState('');
+  // state of scenario type selected, for new scenario creation
+  const handle_scenTypebtn_change = (event) => {
+    set_newScenType_state(event.target.value);
   };
-
-  // Function to handle form submission (you can add your fetch logic here)
-  const handleSubmit = (event) => {
+  const handle_scenName_change = (event) => {
+    set_newScenName_state(event.target.value);
+  };
+  const handle_groupName_change = (event) => {
+    set_newScen_groupName_state(event.target.value);
+  };
+  const handle_createScenario_submit = (event) => {
     event.preventDefault();
     console.log('submit clicked')
     axios.post('/api/scenario_interface', {
-        METHOD:'CREATE',
-        type: scenarioType_state,
-        name: scenarioName_state,
-        group_name: scenGroup_name_state
+      METHOD:'CREATE',
+      type: newScenType_state,
+      name: newScenName_state,
+      group_name: newScen_groupName_state
     })
-    // Perform your fetch request here with scenarioType_state and scenarioName_state
   };
+  
+  // EXISTING SCENARIOS
+  // state of info for currently selected scenario (previously created), from instructor table
+  const [scenarioDetail_state, set_scenarioDetail_state] = useState({})
 
   return (
     <>
+      <div className="group-creation-container">
```

Comment:
added quick&dirty input field element to submit a new group name to /create_group (works)